templates/v2/vault/folder/_card.html.twig

```diff
@@ -2,8 +2,17 @@
     {{ stimulus_controller('ajax-toggle', {'url': path('folder_toggle_visibility', {'id': folder.id}) })|stimulus_target('drag_n_drop', 'folder') }}
     data-id="{{ folder.id }}"
 >
-    <td class="bg-white borderless text-start align-middle  personal-data-card position-relative {{ folder.bPrive ? 'private' : 'shared' }}"
-        {{ stimulus_target('ajax-toggle', 'card') }}
```

Comment:
It's a little heavy solution, I think only this makes the same result :
    `<td class="bg-white borderless text-start align-middle position-relative {{ folder.bPrive ? 'private' : 'shared' }} {{ app.user.isBeneficiaire ? 'personal-data-card' : '' }}"`

Reply:
If it works, you can adapt this on other cards